_jobs/positions/__QA-testing-engineer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: QA Testing & Automation Engineer
+is_fellow: true
+lever_link:
+position: 1
+lede_markdown: We’re looking for skilled manager and mentor to lead engineering practices for City of Austin’s Design, Technology, and Innovation Fellows program.
+---
+
+## Overview
+We are looking for a skilled and passionate QA Testing & Automation expert. As our project load increases, we are increasingly in need of a point person to help streamline our QA process for current and future projects to ensure we stick to a timely and frequent release schedule without compromising on quality and accessibility for Austin residents.
+
+This is a position of tremendous importance that has the potential for large impact inside our organization and across the city services. You will be an active, creative participant in shaping the future of these practices. We're looking for someone passionate and creative about making the best use of our people and resources to deliver the most impact. This could involve a wide range of activities, from helping solidify existing QA, testing, and release protocols to helping write or review unit tests and set up cross browser testing.
```

Comment:
```suggestion
This pivotal position has the potential to leverage impact inside our organization and across the city services. The candidate will be instrumental in shaping the future of digital, accessible, and equitable city services. We're looking for someone passionate, creative, resourceful, civic-minded and impact-driven. This may involve a wide range of activities, from helping solidify existing QA, testing, and release protocols to helping write or review unit tests and set up cross-browser testing.
```

Reply:
+ 100 points for alliteration  !